include/net/fota_download.h

```diff
@@ -36,15 +36,26 @@ extern "C" {
 enum fota_download_evt_id {
 	/** FOTA download progress report. */
 	FOTA_DOWNLOAD_EVT_PROGRESS,
-	/** FOTA download finished. */
+
+	/** FOTA download was completed successfully. */
 	FOTA_DOWNLOAD_EVT_FINISHED,
-	/** FOTA download erase pending. */
+
+	/** Currently deleting downloaded FOTA image (after upgrade success or failure).
```

Comment:
```suggestion
	/** Deleting the downloaded FOTA image (after upgrade success or failure).
```
Do we need "currently"?

Reply:
Hmm, the sentence definitely feels weird without it. How about the rephrasing I am about to push?